src/lib/viewers/text/BoxCSV.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import Grid from 'react-virtualized/dist/es/Grid/Grid';
+import { Grid } from '@box/react-virtualized/dist/es/Grid';
```

Comment:
Do we need the object notation? It looks like the Grid barrel file re-exports `Grid` as its default.

Reply:
Yeah I tried without and the unit tests failed, I think because of the jest mapping to the `commonjs` directory might not have the same exports